packages/amplify_storage_plugin_interface/test/types/DownloadFile/downloadFileOptions_test.dart

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_storage_plugin_interface/src/DownloadFile/DownloadFileOptions.dart';
+import 'package:amplify_storage_plugin_interface/src/Storage/StorageAccessLevel.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('DownloadFileOptions can be serialized as a Map', () async {
+    DownloadFileOptions options =
+        DownloadFileOptions(accessLevel: StorageAccessLevel.private);
+    var serializedResult = options.serializeAsMap();
+    expect(serializedResult, isInstanceOf<Map>());
+    expect(serializedResult['accessLevel'], 'private');
+  });
```

Comment:
This testing strategy would ensure that the expected fields are there in the serialized map but not absence of anything else that may end up there. We should assert the whole serialized result against an expected (overall) serialized value.